src/controllers/v1/transaction/index.ts

```diff
@@ -21,17 +22,26 @@ export class TransactionController extends Controller {
 
   /**
    * Returns the details of all transactions.
-   * @summary List all transactions
+   * @summary List transactions
    * @Query apiType lists all transactions by that type
    */
   @Response<BadRequest>(400, 'Request was invalid')
   @Response<NotFound>(404, 'Item not found')
   @Get('/')
   public async getAllTransactions(
     @Query() apiType?: TransactionApiType,
-    @Query() status?: TransactionState
+    @Query() status?: TransactionState,
```

Comment:
why not transaction status? not blocking as it was already implemented like that

Reply:
We're already on the `/transaction` path so the word would be redundent I think